packages/wonder-stuff-ci/src/release-utils.ts

```diff
@@ -0,0 +1,70 @@
+import execProm from "./exec-prom";
```

Comment:
I would rename this file to `mobile-release-utils.ts`. That being said, this logic is rather specific to our release processes!